code/datums/emotes.dm

```diff
@@ -0,0 +1,165 @@
+#define EMOTE_VISIBLE 1
+#define EMOTE_AUDIBLE 2
+
+/datum/emote
+	var/key = "" //What calls the emote
+	var/key_third_person = "" //This will also call the emote
```

Comment:
This should be a list of aliases, not a hardcoded "third person key!"

Reply:
I agree